drivers/wifi/nrf700x/osal/fw_if/umac_if/src/tx.c

```diff
@@ -849,9 +849,9 @@ unsigned int tx_buff_req_free(struct wifi_nrf_fmac_dev_ctx *fmac_dev_ctx,
 		} else if (tx_done_q == 2) {
 			tx_done_q = WIFI_NRF_FMAC_AC_BE;
 		} else if (tx_done_q == 4) {
```

Comment:
Although not part of the changes but please replace magic numbers like 2,4 etc with macros.

Reply:
This can be a helper function.